ads/google/a4a/test/test-traffic-experiments.js

```diff
@@ -71,27 +71,26 @@ describe('all-traffic-experiments-tests', () => {
 
     it('handles empty experiments list', () => {
       // Opt out of experiment.
-      // TODO(tdrl): remove the direct access to AMP_CONFIG
-      sandbox.win.AMP_CONFIG['testExperimentId'] = 0.0;
+      toggleExperiment(sandbox.win, 'testExperimentId', false, true);
```

Comment:
I don't believe it actually impacts anything but I do not believe we expect our experiments to be durable past a given page so that last parameter here should actually be false.